apps/cricket-score-frontend/src/components/match/NotFoundDisplay.tsx

```
@@ -0,0 +1,26 @@
+    >
+      <div className="w-16 h-16 mx-auto bg-muted/40 backdrop-blur-sm rounded-full flex items-center justify-center border-2 border-muted/60 relative shadow-xl">
+        <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8 text-muted-foreground" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+          <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M9 12h6m-6 4h6m2 5H7a2 2 0 01-2-2V5a2 2 0 012-2h5.586a1 1 0 01.707.293l5.414 5.414a1 1 0 01.293.707V19a2 2 0 01-2 2z" />
+        </svg>
+      </div>
+      <h2 className="mt-4 text-2xl font-bold text-foreground">Match Not Found</h2>
+      <p className="mt-2 text-muted-foreground">We couldn&apos;t find the details for this match.</p>
+      <button
+        onClick={() => window.history.back()}
+        className="btn btn-primary mt-6 px-5 py-2.5"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)